sdk/cosmosdb/cosmos/README.md

```diff
@@ -183,6 +183,18 @@ for (const city of resources) {
 
 For more information on querying Cosmos DB databases using the SQL API, see [Query Azure Cosmos DB data with SQL queries][cosmos_sql_queries].
 
+## Error Handling
```

Comment:
I love this section! We should think about how to advertise what sorts of Errors are thrown in other SDKs as well.

/cc @willmtemple @bterlson @diberry

Reply:
@witemple-msft — just making sure I’ll have a notif for this on my MS acct. when I’m back after the holidays.